pkgs/php/8.2.nix

```diff
@@ -0,0 +1,65 @@
+{ prev, mkPhp, php82 ? null}:
+
+let
+  base = mkPhp {
+    version = "8.2.0alpha2";
+    sha256 = null;
+
+    phpAttrsOverrides = attrs: {
+      src = php82;
```

Comment:
I do not see much benefit in specifying it in `flake.nix` when it is not easily updateable, just use `fetchurl` here.

Reply:
Right, fixed !